### PR TITLE
TC 01.4.1_04 Header/Verify Shopping Cart Icon

### PR DESCRIPTION
**Pull request by RomanBurlaka78**
[TC 01.4.1_04 Header/Verify Shopping Cart Icon](https://trello.com/c/XEhfX1E6)

### CODE REVIEW

**Review by SerhiiPetrykin — APPROVED** [review]
Delete an empty string (52)

**Comment by RomanBurlaka78** [comment]
ok.

File: tests/headerPanel.spec.js [hunk 1]
```diff
@@ -51,3 +50,17 @@ test.describe('headerPanel verify searchInput', ()=>  {
     })
 })
 
+test.describe('headerPanel / Verify shoping cart', ()=>  {
```
Comment (SvetlanaKanel) [inline tests/headerPanel.spec.js:53]: Why do you decide to add a new describe and beforeEach? This file already had the describe and beforeEach on the 3 and 5 lines.

**Review by mlimau — APPROVED** [review]

File: tests/headerPanel.spec.js (hunk 1 above)
Comment (RomanBurlaka78) [reply tests/headerPanel.spec.js:53]: This is not my js file, and i dont want to make  problem with outhers tests

**Pull request merged**